### ISSUE TITLE
Fix tabs to be persistent on scroll

### ISSUE THREAD

**Issue by nguyetv** [joincivil/Civil#1072]

**Comment by nguyetv** [comment]
![screen shot 2019-03-06 at 2 51 47 pm](https://user-images.githubusercontent.com/6794566/53909503-d6000480-401f-11e9-8cac-2ab23772ab8d.png)

**Comment by am9u** [comment]
@nguyetv To clarify, by "persistent" do you mean that:

- Once the user scrolls past the hero image, that the top of the "Approved | In Progress | Rejected" tab bar should be set at a fixed position right under the top navigation
- The page content should scroll freely underneath the fixed tab navigation

**Comment by nguyetv** [comment]
@am9u yes, that is correct. Thanks for clarifying questions, and clearer bullet descriptions. I will try to make better tickets next time <3
